tests/test-cases/Group1-Docker-Commands/1-39-Docker-Stats.robot

```diff
@@ -0,0 +1,76 @@
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation   Test 1-39 - Docker Stats
+Resource        ../../resources/Util.robot
+Suite Setup     Install VIC Appliance To Test Server
+Suite Teardown  Cleanup VIC Appliance On Test Server
+
+*** Test Cases ***
+Create test containers
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull busybox
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d --name stresser busybox /bin/top
+    Should Be Equal As Integers  ${rc}  0
+    Set Environment Variable  STRESSED  ${output}
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create --name stopper busybox /bin/top
+    Should Be Equal As Integers  ${rc}  0
+    Set Environment Variable  STOPPER  ${output}
+
+Stats No Stream
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} stats --no-stream
+    Should Be Equal As Integers  ${rc}  0
+    ${output}=  Get Line  ${output}  1
```

Comment:
-1 grabs the last line of output. so in case for some reason there was a hangover container or something it would be safer as it would always grab the last line